docs/serving/samples/hello-world/helloworld-nodejs/README.md

```diff
@@ -93,7 +93,7 @@ recreate the source files from this folder.
    # Copy application dependency manifests to the container image.
    # A wildcard is used to ensure both package.json AND package-lock.json are copied.
    # Copying this separately prevents re-running npm install on every code change.
-   COPY package*.json ./
+   COPY package.json package*.json ./
```

Comment:
Really? COPY doesn't recognized package.json when using the wildcard? If so, please update the comment.

Reply:
Nice catch. I just tested out, wildcard works for package.json. Updated